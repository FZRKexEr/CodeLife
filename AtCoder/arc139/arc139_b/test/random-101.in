100
9 2 14 20 9 19
53 18 12 1 8 9
78 13 2 13 18 13
12 10 2 9 10 4
90 13 2 10 19 3
89 16 19 9 12 19
56 19 17 6 5 2
55 16 4 3 16 18
23 17 14 7 6 8
72 11 13 3 9 20
23 4 3 9 15 18
3 4 11 18 14 12
86 6 19 12 15 15
85 6 12 8 14 19
53 20 7 2 16 2
60 5 17 6 14 15
26 12 16 19 20 9
34 8 10 14 1 9
70 20 17 5 6 4
9 19 6 9 2 11
81 1 11 19 17 15
83 20 13 20 5 14
34 7 14 18 9 12
59 1 11 4 14 9
90 9 14 11 10 6
23 13 15 4 8 8
67 13 12 5 10 14
91 4 4 10 3 5
15 15 15 20 13 8
63 1 8 13 18 10
15 9 4 19 8 13
90 17 17 14 18 17
47 11 5 14 12 11
74 7 19 10 3 13
88 9 11 1 2 12
98 19 10 8 5 14
34 11 15 2 4 10
25 9 10 3 2 17
59 20 15 1 19 17
22 10 3 6 17 2
35 16 16 20 16 4
18 1 7 17 20 18
88 6 8 8 1 5
25 5 2 17 17 13
23 16 15 4 1 3
34 15 19 13 14 2
11 8 6 4 11 5
89 8 6 17 16 19
98 14 4 17 1 14
63 12 3 11 17 1
30 19 19 2 14 10
61 8 10 11 5 10
53 2 2 6 9 17
39 18 13 20 3 2
93 17 2 4 3 12
39 10 3 3 14 19
91 16 10 8 4 3
6 18 5 8 7 13
32 17 18 16 5 13
47 8 1 14 8 8
63 8 6 4 20 18
72 12 18 6 14 3
35 6 17 1 18 15
9 11 8 4 18 15
3 18 1 3 9 1
16 8 3 15 4 20
38 8 2 8 2 1
89 7 2 12 8 9
19 13 8 13 1 9
88 16 3 1 12 12
53 7 11 6 13 1
58 11 1 7 12 17
95 13 11 7 10 5
66 15 8 1 11 13
29 10 9 11 14 1
48 14 14 19 12 8
32 12 17 13 3 13
61 18 4 19 7 15
2 10 17 13 6 9
59 8 11 2 11 12
8 7 17 19 13 8
47 6 11 19 5 9
66 16 12 6 6 18
19 9 18 9 2 7
27 18 6 19 4 10
84 1 5 3 20 15
68 7 14 7 1 10
59 10 5 11 8 17
61 12 11 10 7 7
58 8 2 15 9 2
48 20 9 9 7 16
72 4 5 1 8 5
81 19 13 13 18 3
14 1 6 7 4 1
12 19 15 6 1 3
80 19 12 12 8 19
36 5 20 10 11 8
81 12 3 2 6 20
95 2 19 11 6 12
23 2 19 14 4 19
