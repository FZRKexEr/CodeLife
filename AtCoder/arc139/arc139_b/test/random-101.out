56
30
108
14
132
81
18
69
48
72
93
54
123
141
22
83
78
12
26
38
173
80
132
61
72
40
60
96
8
161
59
155
79
51
25
49
14
25
57
20
68
104
22
85
29
16
23
193
7
21
32
56
240
6
47
69
31
21
237
96
162
12
31
19
3
8
11
126
57
68
29
81
62
50
25
138
55
93
26
68
32
39
36
13
71
252
44
101
45
58
64
72
57
10
72
80
51
54
60
27
